packages/web-app-files/src/mixins/actions/copy.js

```diff
@@ -53,9 +56,19 @@ export default {
   },
   methods: {
     ...mapActions('Files', ['copySelectedFiles']),
-
-    $_copy_trigger() {
+    ...mapMutations('Files', ['SET_FILE_SELECTION']),
+    $_copy_trigger({ resources }) {
+      this.SET_FILE_SELECTION(resources)
```

Comment:
Can we change the `copySelectedFiles` action in the store instead? You can add an optional param for the resources to be copied/moved. Feels weird needing to set the selection in the store first (which could have other side effects).